ssh_check/README.md

```diff
@@ -40,7 +40,7 @@ The SSH/SFTP check is included in the [Datadog Agent][1] package, so you don't n
 See [metadata.csv][5] for a list of metrics provided by this check.
 
 ### Events
-The SSH Check check does not include any event at this time.
+The SSH Check check does not include any events at this time.
```

Comment:
Check check? Should that word be use twice?

Reply:
nop